playbooks/operationalizing/cloudforms_networking.adoc

```diff
@@ -55,6 +55,28 @@ Also configured in the LTM are two sets of load balancing pools. Load balancing
 
 Once these two components have been defined, the virtual server associated to port 8443 can include the load balancing pool containing the set of master instances on port 8443. The virtual server associated to port 5000 can include the load balancing pool containing the set of node instance that contain running instances of the integrated HAProxy router to pass traffic to the Hawkular metrics services within the OpenShift cluster.
 
+===== Haproxy Configuration
+Software load balancer(haproxy) can be used which Openshift provide if there is no F5. In this case, Haproxy play a role in both load balancer for multiple masters and a reverse  ↳proxy for hawkular metrics. Following haproxy configuration is for hawkular metrics. (It should be added at the end of /etc/haproxy/haproxy.cfg file)
+
+[source]
+----
+frontend hawkular-metrics
+   bind *:5000
+   default_backend hawkular-metrics
+   mode tcp
+   option tcplog
+
+backend hawkular-metrics
+    balance source
+    mode tcp
+    option tcplog
+
+# Use physical ip for infra node
+    server infra1 10.x.x.x:443 check
+
+# if you have multiple routers, you have to add all of them.
+#   server infra2 /10.x.x.y:443 check
```

Comment:
Instead of multiple routers, do we want to say multiple nodes with routers deployed?